alarm/models.py

```diff
@@ -16,6 +17,211 @@ class Coin(models.Model):
+                   'lskusdt',
+                   'tusdgbp',
+                   'btctry',
+                   'ltctry',
+                   'ftmtusd',
+                   'xlmngn',
+                   'bchtry',
+                   'ftmusdt',
+                   'bchngn',
+                   'ftmbnb',
+                   'hotbnb',
+                   'usdtry',
+                   'usdttry',
+                   'dogeusdt',
+                   'dogebnb',
+                   'ethtry',
+                   'duskbtc',
+                   'dusketh',
+                   'phbusdt',
+                   'phbbnb',
+                   'tusdbnb',
+                   'eosbusd',
+                   'neoeth',
+                   'iotabnb',
+                   'wavesbnb',
+                   'wavesbtc',
+                   'stratbtc',
+                   'stratusd',
+                   'adxusdt',
+                   'adaeur',
+                   'kavausdt',
+                   'trxngn',
+                   'kavaeur',
+                   'usdceur',
+                   'xrpeur',
+                   'bcheur',
+                   'ltceur',
+                   'btcgbp',
+                   'ethgbp',
+                   'xrpgbp',
+                   'bnbpax',
+                   'bchgbp',
+                   'ltcgbp',
+                   'usdcpax',
+                   'linketh',
+                   'usdcusdt',
+                   'waveseth',
+                   'linkbtc',
+                   'atometh',
+                   'usdcngn',
+                   'dasheth',
+                   'dgbusdt',
+                   'btczrx',
+                   'dgbbnb',
+                   'maticngn',
+                   'matictry',
+                   'maticdai',
+                   'maticrub',
+                   'btcuah',
+                   'usdcuah',
+                   'xrptry',
+                   'btctry',
+                   'xrpars',
+                   'xlmeur',
+                   'eosgbp',
+                   'dogeeur',
+                   'xlmbnb',
+                   'ethuahe',
+                   'zileth',
+                   'btceur',
+                   'zilbnb',
+                   'usdtzar',
+                   'xrpbullusdt',
+                   'xrpbearusdt',
+                   'tusdbtc',
+                   'etcbtc',
+                   'etcbnb',
+                   'ltcbullusdt',
+                   'ltcbearusdt',
+                   'ethtusd',
+                   'daiusdt',
+                   'cvcusdt',
+                   'cvceth',
+                   'btcngnb',
+                   'ethngn',
+                   'dntusdt',
+                   'dnteth',
+                   'sntusdt',
+                   'senusdt',
+                   'bnteth',
+                   'trxngnb',
+                   'trxtry',
+                   'vibeth',
+                   'vibusdt',
+                   'mdabtc']
+        return self.coin_abbreviation in symbols
```

Comment:
Can we get this list only when need, from binance? What if binance will add support for new coins, do we need to add them manually?

Reply:
I had an idea to query a link that returns a sheet of coin names, but so far I can't get it to work. If this idea is relevant, I'll keep working on it.